roles/mev_boost/defaults/main.yml

```diff
@@ -0,0 +1,32 @@
+mev_boost_user: mev_boost
+mev_boost_datadir: /data/mev_boost
+
+mev_boost_cleanup: false # when set to "true" it will remove the container(s)
+
+################################################################################
+##
+## mev_boost container configuration
+##
+################################################################################
+mev_boost_container_name: mev_boost
+mev_boost_container_image: flashbots/mev-boost:1.5.0
+mev_boost_container_env: {}
+mev_boost_container_ports:
+  - "127.0.0.1:{{ mev_boost_container_port }}:{{ mev_boost_container_port }}"
+
+mev_boost_network: goerli
```

Comment:
I would prefer if we would remove the following vars:

- `mev_boost_network`
- `mev_boost_timeout`
- `mev_boost_relays`. 

I think we don't need to abstract these away. They're only being used on the CLI flags and for that it's better if we directly set whatever is needed there. 

Also, I think it would be nice if we would default `mev_boost_container_command` to a default simple mainnet configuration. This would mean leaving everything as it is, and just adding the flags that are required for it to run against mainnet. The reasoning for this is that all the other charts also have mainnet defaults and try to not setup any CLI flags that aren't strictly necessary. 

My proposal would be to just keep it like: 

```yaml
mev_boost_container_command:
  - --relays=https://0xac6e77dfe25ecd6110b8e780608cce0dab71fdd5ebea22a16c0205200f2f8e2e3ad3b71d3499c54ad14d6c21b41a37ae@boost-relay.flashbots.net
  ```

Reply:
Update: I noticed that defining the port is also missing. So if we additionaly have a server port var, we should have the following args:

```yaml
mev_boost_container_command:
  - --addr=0.0.0.0:{{ mev_boost_server_port }} 
  - --relays=https://0xac6e77dfe25ecd6110b8e780608cce0dab71fdd5ebea22a16c0205200f2f8e2e3ad3b71d3499c54ad14d6c21b41a37ae@boost-relay.flashbots.net

Reply:
yea i was initially thinking the `network` atleast is set as a global, but if we aren't going down that route then its easier to explicitly define it anyway